liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/SpringBootDevTest.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2019, 2022.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package net.wasdev.wlp.test.dev.it;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.util.Scanner;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import static io.openliberty.tools.common.plugins.util.BinaryScannerUtil.*;
+
+public class SpringBootDevTest extends BaseDevTest {
+
+   @BeforeClass
+   public static void setUpBeforeClass() throws Exception {
```

Comment:
You have to do more like `mvn package liberty:dev`.  

That said.. I think it might be a bad idea to include this test.. since someone later might assume it implies that we have support for dev mode in Spring Boot which we don't.

This would fit better as a test of 'run'.  You still have to do a package with 'run'  (so `mvn package liberty:run`) but you don't have to use loose apps, and so it makes more sense than dev (since neither loose apps in general nor dev work with SB and LMP currently).

Reply:
^^ What Scott said :-)